tests/test_psf.py

```diff
@@ -347,6 +349,117 @@ def test_chromatic_psf(self):
+            },
+            'stamp': {
+                'type': 'LSST_Silicon',
+
+                'fft_sb_thresh': 2.e5,   # When to switch to fft and a simpler PSF and skip silicon
+                'max_flux_simple': 100,  # When to switch to simple SED
+
+                'airmass': self.opsim_data['airmass'],
+                'rawSeeing': self.opsim_data['rawSeeing'],
+                'band':  self.opsim_data['band'],
+
+                'diffraction_psf': {
+                    'enabled': False,
+                    'exptime': 30,
+                    'azimuth': "0 deg",
+                    'altitude': "60 deg",
+                    'rotTelPos': "0 deg",
+                },
+                'det_name': 'R22_S11',
+                'world_pos':  {
+                    'type': 'RADec',
+                    'ra' : '@input.atm_psf.boresight.ra',
+                    'dec' : '@input.atm_psf.boresight.dec',
+                },
+            },
+            'image' : {
+                'size': 64,
+                'random_seed': 1234,
+                'wcs': {
+                    'type' : 'Tan',
+                    'dudx' : 0.2,
+                    'dudy' : 0.,
+                    'dvdx' : 0.,
+                    'dvdy' : 0.2,
+                    'ra' : '@input.atm_psf.boresight.ra',
+                    'dec' : '@input.atm_psf.boresight.dec',
+                },
+                'bandpass': {
+                    'file_name': 'LSST_r.dat',
+                    'wave_type': 'nm',
+                    'thin': 1.e-4,
+                },
+                'noise': {'type': 'Poisson'},
+            }
+        }
+
+        # First make a reference image, using photon shooting
+        config1 = galsim.config.CopyConfig(config)
+        ref_img = galsim.config.BuildImage(config1)
+
+        # Repeat with an object bright enough to switch to FFT
+        config['gal']['flux'] = 1.e8
+        with CaptureLog() as cl:
+            img = galsim.config.BuildImage(config, logger=cl.logger)
+        #print(cl.output)
+        assert 'Check if we should switch to FFT' in cl.output
+        assert 'Yes. Use FFT for this object.' in cl.output
+
+        print('Peak of reference PSF (flux=1.e5): ',ref_img.array.max())
+        print('Peak of FFT PSF (flux=1.e8): ',img.array.max())
+        print('FWHM of reference PSF: ',ref_img.view(scale=0.2).calculateFWHM())
+        print('FWHM of FFT PSF: ',img.view(scale=0.2).calculateFWHM())
+        print('Rmom of reference PSF: ',ref_img.view(scale=0.2).calculateMomentRadius())
+        print('Rmom of FFT PSF: ',img.view(scale=0.2).calculateMomentRadius())
+
+        # The FFT image is about 10^3 x brighter than the reference image.
+        # Scale it down to make it easier to compare to the reference image.
+        img /= 1.e3
+
+        # Peaks should now be similar
+        np.testing.assert_allclose(ref_img.array.max(), img.array.max(), rtol=0.05)
+
+        # The sizes should also be pretty close
```

Comment:
Looks like CI failed here pretty dramatically.  Do we still expect the RNGs to be close to platform independent?  Other than this, PR all looks okay to me.

Reply:
Weird.  I certainly expected the random numbers to be portable.  We even have unit tests in GalSim to verify that the same random values are produced on all systems.  Hm...  I'll try running on some other systems besides my laptop to see if I can see what is going on with that.

Reply:
I'm a little stymied by this.  I can't reproduce the failure on any system I've tried.  USDF, Perlmutter, and BNL all give exactly the same numbers as my laptop, and thus they all pass.  I have no idea why GitHub CI is getting different values.

Reply:
OK, I more or less figured it out.  It turns out that on GitHub, the image position was ending up as galsim.PositionD(5.636331611506773e-11,-2.8624992133171645e-11) rather than (0,0).  This ended up causing a different centering of the stamp, which would be fine for normal things where it gets dropped onto a larger image in the right place.  But in my test, my size estimators were centered on the image center, which wasn't the center of the object. 
Anyway, this led me to adjust the stamp builder to allow users to input image_pos, rather than world_pos, so I can give (0,0) explicitly, rather than have the wcs calculate it.  This is something we should really have had already to better match how the default stamp builder works, so that's a bonus.